db/migrate/20250207062248_add_exchange_to_import_rows.rb

```diff
@@ -0,0 +1,5 @@
+class AddExchangeToImportRows < ActiveRecord::Migration[7.2]
+  def change
+    add_column :import_rows, :exchange, :string
```

Comment:
This is outdated I believe.  I think we can remove this column and all related code now?